apps/core/Search.js

```diff
@@ -1,58 +1,82 @@
 // @flow
 
 import * as React from 'react';
-import { StyleSheet, View, Button } from 'react-native';
-import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
-import { Text } from '@kiwicom/margarita-components';
+import { StyleSheet, View, Button, TextInput } from 'react-native';
 
 import Routes from './config/routes';
-import { type SearchQueryResponse } from './__generated__/SearchQuery.graphql';
 
 type Props = Object;
+type State = Object;
```

Comment:
We will need eventually to make these types more precise.

Reply:
Definitely, currently I treat it as simplified version which will be greatly expanded in next iteration. When most of the states and props will be expanded and precisely defined.

Reply:
For now I just at least updated state props and after we update navigation, I'll define even props more precisely in next iteration.